src/global.css

```
@@ -1,325 +1,373 @@
-body::-webkit-scrollbar {
-  width: 6px;
-  height: 6px;
+/* Horizontal scrollbar specific styling */
+.custom-scrollbar::-webkit-scrollbar-horizontal {
+    height: 8px;
 }
 
-body::-webkit-scrollbar-track {
-  background: rgba(0, 0, 0, 0.1);
-  border-radius: 3px;
+.custom-scrollbar::-webkit-scrollbar-horizontal:track {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class track 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)